consensus/src/runway/mod.rs

```diff
@@ -171,18 +172,67 @@ impl<'a, H: Hasher> RunwayStatus<'a, H> {
     }
 }
 
+fn short_report(rounds_behind: Round, missing_coords: usize) -> String {
+    if rounds_behind <= 2 && missing_coords == 0 {
```

Comment:
nit: I think match here would be more readable, feel free to ignore

Reply:
Found a tiny bug in 3rd if condition, so maybe you're right